src/components/ha-sidebar.ts

```diff
@@ -255,12 +258,11 @@ class HaSidebar extends LitElement {
             : "Home Assistant"}
         </div>
       </div>
-      <paper-listbox
-        attr-for-selected="data-panel"
+      <mwc-list
+        activatable
+        @selected=${this._selectItem}
         class="ha-scrollbar"
-        .selected=${hass.panelUrl}
```

Comment:
You should set `activated` on the panel that is active

Reply:
It seems to already do that by default.

![image](https://user-images.githubusercontent.com/1480827/96620656-93ef0980-12bc-11eb-91a6-f75b489d7d22.png)